tests/drum/run-drum-tests-in-container.sh

```diff
@@ -24,15 +24,39 @@ echo
 echo "--> Building wheel"
 echo
 make clean && make
-CMRUNNER_WHEEL=$(find dist/datarobot_drum*.whl)
-CMRUNNER_WHEEL_REAL_PATH=$(realpath $CMRUNNER_WHEEL)
+DRUM_WHEEL=$(find dist/datarobot_drum*.whl)
+DRUM_WHEEL_FILENAME=$(basename $DRUM_WHEEL)
+DRUM_WHEEL_REAL_PATH=$(realpath $DRUM_WHEEL)
 
-echo "CMRUNNER_WHEEL_REAL_PATH: $CMRUNNER_WHEEL_REAL_PATH"
+echo "DRUM_WHEEL_REAL_PATH: $DRUM_WHEEL_REAL_PATH"
 
 echo
 echo "--> Installing wheel"
 echo
-pip install "${CMRUNNER_WHEEL}[R]"
+pip install "${DRUM_WHEEL}[R]"
+popd
+
+
+# Change every environment Dockerfile to install freshly built DRUM wheel
+WITH_R=""
+pushd $GIT_ROOT/public_dropin_environments
+DIRS=$(ls)
+for d in $DIRS
+do
```

Comment:
As I use our public envs in tests, I need to replace dependency like:
datarobot-drum==1.4.16 with a wheel compiled in the current tests session.

I do the same in the functional tests for a long time.